File: LeetCode Practice Problems/961- N-Repeated Element in Size 2N Array.cpp
class Solution {
public:
    int repeatedNTimes(vector<int>& A) {
        set<int> s;
        for(auto i : A)
        {
            if(s.find(i) != s.end())
                return i;
            s.insert(i);
        }
        
        return A[0];
    }
};

File: LeetCode/LeetCode Practice Problems/961-N-Repeated Element in Size 2N Array.java
class Solution {
    public int repeatedNTimes(int[] nums) {
        int cnt[] = new int[11000];
        int ans = 0;
        int n = nums.length;
        
        for(int num: nums) {
            cnt[num]++;
            if(cnt[num] == n/2) {
                ans = num;
            }
        }
        
        return ans;
    }
}
